### PR TITLE
chore: translate more csharp docs

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

File: website/docs/cdktf/concepts/constructs.mdx [hunk 1]
```diff
@@ -358,3 +404,7 @@ You can search the Amazon Web Services (AWS) [Construct Hub](https://constructs.
 ## Write and Publish Constructs
 
 You can write construct classes in any language and distribute them to other users. Refer to [Construct Design](/cdktf/develop-custom-constructs/construct-design) and [Construct Publishing and Distribution](/cdktf/develop-custom-constructs/publishing-and-distribution) for details.
+
+```
```
Comment (mutahhir) [inline website/docs/cdktf/concepts/constructs.mdx:408]: This is a stray code block

File: examples/csharp/documentation/MyTerraformStack.csproj [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp6.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="HashiCorp.Cdktf" Version="0.0.0" />
+    <PackageReference Include="HashiCorp.Cdktf" Version="0.14.3" />
```
Comment (mutahhir) [inline examples/csharp/documentation/MyTerraformStack.csproj:9]: I suppose this was unintentional for testing?

**Review by mutahhir — APPROVED** [review]
Nice. Approving, just one minor comment.

**Comment by team-tf-cdk** [comment]
This PR exceeds the recommended size of 1000 lines. Please make sure you are NOT addressing multiple issues with one PR. Note this PR might be rejected due to its size.

**Comment by team-tf-cdk** [comment]
This PR exceeds the recommended size of 1000 lines. Please make sure you are NOT addressing multiple issues with one PR. Note this PR might be rejected due to its size.

**Pull request merged**